extensions/amp-list/0.1/amp-list.js

```diff
@@ -552,17 +554,15 @@ export class AmpList extends AMP.BaseElement {
       scheduleNextPass();
       current.rejecter();
     };
+    const renderTemplate = this.ssrTemplateHelper_.renderTemplate(
+        this.element, current.data);
     if (this.ssrTemplateHelper_.isSupported()) {
-      const html = /** @type {string} */ (current.data);
-      this.templates_.findAndSetHtmlForTemplate(this.element, html)
-          .then(result => this.updateBindings_([result]))
+      renderTemplate.then(result => this.updateBindings_([result]))
```

Comment:
Looks like you can also dedupe the `updateBindings_` call and callback invocations.